docs/docs/tutorials/orca_k8s/orca_k8s_tutorial.md

```diff
@@ -0,0 +1,563 @@
+* `num_nodes`: an Integer that specifies the number of executors (default to be `1`).
+* `driver_cores`: an Integer that specifies the number of cores for the driver node (default to be `4`).
+* `driver_memory`: a String that specifies the memory for the driver node (default to be `"1g"`).
+* `extra_python_lib`: a String that specifies the path to extra Python packages, one of `.py`, `.zip` or `.egg` files (default to be `None`).
+* `penv_archive`: a String that specifies the path to a packed Conda archive (default to be `None`).
+* `jars`: a String that specifies the path to needed jars files (default to be `None`).
+* `conf`: a Key-Value format to append extra conf for Spark (default to be `None`).
+
+__Note__: 
+* All arguments __except__ `cluster_mode` will be ignored when using `spark-submit` to submit and run Orca programs.
+
+After the Orca programs finished, you should call `stop_orca_context` at the end of the program to release resources and shutdown the underlying distributed runtime engine (such as Spark or Ray).
+```python
+from bigdl.orca import stop_orca_context
+
+stop_orca_context()
+```
+
+For more details, please see [OrcaContext](https://bigdl.readthedocs.io/en/latest/doc/Orca/Overview/orca-context.html).
+
+## 1.2 K8s Client&Cluster
+The difference between k8s-client and k8s-cluster is where you run your Spark driver. 
+
+For k8s-client, the Spark driver runs in the client process (outside the K8s cluster), while for k8s-cluster the Spark driver runs inside the K8s cluster.
+
+Please see more details in [K8s-Cluster](https://spark.apache.org/docs/latest/running-on-kubernetes.html#cluster-mode) and [K8s-Client](https://spark.apache.org/docs/latest/running-on-kubernetes.html#client-mode).
+
+
+## 1.3 Load Data from Network File Systems (NFS)
+When you are running programs on K8s, please load data from a network file system (NFS) instead of the local file system.
+
+After mounting the NFS into BigDL container (see __Section 2.2 & 4__), the Fashion-MNIST example could load data from NFS the same as loading from a local file system.
+
+```python
+import torch
+import torchvision
+import torchvision.transforms as transforms
+from bigdl.orca.data.file import get_remote_file_to_local
+
+def train_data_creator(config, batch_size):
+    transform = transforms.Compose([transforms.ToTensor(),
+                                    transforms.Normalize((0.5,), (0.5,))])
+
+    trainset = torchvision.datasets.FashionMNIST(root="/bigdl/nfsdata/dataset", train=True, 
+                                                 download=False, transform=transform)
+
+    trainloader = torch.utils.data.DataLoader(trainset, batch_size=batch_size, shuffle=True, num_workers=0)
+    return trainloader
+```
+
+# 2. Create & Lunch BigDL K8s Container 
+## 2.1 Pull Docker Image
+Please pull the prebuilt BigDL `bigdl-k8s` image from [Docker Hub](https://hub.docker.com/r/intelanalytics/bigdl-k8s/tags) as follows:
```

Comment:
Do we need to mention that users other recommended to pull the image on all nodes otherwise they may need to wait for the image pull at runtime?